.gitignore

```diff
@@ -180,3 +180,6 @@ TempTypeSpecFiles/
 
 # Common toolchain intermediate files
 temp
+
+# Runsettings files
```

Comment:
Please remove.   We have teams that intentionally include `runsettings` files to align their testing configuration across their development team.

Reply:
Fixed.